src/stories/assets/form/occurrenceReportingForm.json

```diff
@@ -26830,7 +26831,24 @@
     "accountName" : "starr@engisafe.com",
     "firstName" : "Ringo",
     "lastName" : "Starr"
-  } ],
+  },
```

Comment:
Not sure why we need this question here.

Reply:
@LaChope is this resolved?

Reply:
I do not think it is needed, I just copy/pasted the form. It is done now, I removed it.

Reply:
Ok I meant something else but I guess it is clear now ..., resolving this